requirements.in.txt

```diff
@@ -1,6 +1,7 @@
 git+https://github.com/ckan/ckan.git@ckan-2.9.3#egg=ckan
 -e git+https://github.com/GSA/ckanext-googleanalyticsbasic#egg=ckanext-googleanalyticsbasic
 -e git+https://github.com/GSA/ckanext-usmetadata.git@main#egg=ckanext-usmetadata
+-e git+https://github.com/GSA/ckanext-dcat_usmetadata.git@main#egg=ckanext-dcat_usmetadata
```

Comment:
Are we going to revert this if we get on pypi? Should we wait until it's deployed?

Reply:
I have to update this.  Waiting to see if dcat_usmetadata deploys to pypi first though 😅

Reply:
It made it to pypi! https://pypi.org/project/ckanext-dcat-usmetadata/

Reply:
Running tests locally with a new build and then will push.  After this is done, I don't think I know what to do with my life 😆